### PR TITLE
[DGPFUS-37] Database Design For A Transportation Company

### PR DESCRIPTION
**Pull request by ChijiokeOrabueze**
In this PR,

- I wrote sql code for a transportation company database design.

### CODE REVIEW

File: submissions/chijiokeOrabueze/database2/database2.sql [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+CREATE DATABASE travel_sample;
+
+-- specifying database to use
+USE travel_sample;
+
+
+-- creating passengers table
+CREATE TABLE passengers (
+	id INT NOT NULL AUTO_INCREMENT,
+    full_name VARCHAR(100) NOT NULL,
```
Comment (ToluEnny24) [inline submissions/chijiokeOrabueze/database2/database2.sql:11]: 100 characters is too smalll, change to 250

File: submissions/chijiokeOrabueze/database2/database2.sql [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    sex VARCHAR(10) NOT NULL,
+    age VARCHAR(20),
+    PRIMARY KEY (id)
+);
+
+-- creating passengers_details table
+CREATE TABLE passengers_details (
+	id INT NOT NULL AUTO_INCREMENT,
+    passengers_id INT NOT NULL,
+    passenger_class INT NOT NULL,
```
Comment (ToluEnny24) [inline submissions/chijiokeOrabueze/database2/database2.sql:21]: ENUM 1, 2, 3

**Review by ToluEnny24 — CHANGES_REQUESTED** [review]
Well done @ChijiokeOrabueze Please effect the requested changes

File: submissions/chijiokeOrabueze/database2/database2.sql (hunk 2 above)
Comment (ChijiokeOrabueze) [reply submissions/chijiokeOrabueze/database2/database2.sql]: Done @ToluEnny24

**Review by ToluEnny24 — APPROVED** [review]

**Pull request merged**